src/System.Security.AccessControl/src/System/Security/AccessControl/PrivilegeNotHeldException.cs

```diff
@@ -29,9 +31,15 @@ public PrivilegeNotHeldException(string privilege, Exception inner)
             _privilegeName = privilege;
         }
 
+        private PrivilegeNotHeldException(SerializationInfo info, StreamingContext context) : base(info, context)
+        {
+            _privilegeName = info.GetString(nameof(PrivilegeName));
```

Comment:
Any reason you're using nameof and not raw string? Not sure we're consistent about this. I guess it doesn't matter as nobody will rename it. Maybe ideally we 'd do it in all cases.

Reply:
I used nameof here because it references a public member that won't be renamed unintentionally.